### PR TITLE
Aliases for git switch and restore

### PR DESCRIPTION
**Pull request by jjanvier**
With git 2.23, the `checkout` command has been divided into `switch` and `restore`. Which is much more clear. 

This PR creates aliases for those 2 commands.

More info here https://github.blog/2019-08-16-highlights-from-git-2-23/.

### CODE REVIEW

**Comment by anaelChardan** [comment]
Thanks @jjanvier, :+1:

**Comment by jjanvier** [comment]
Could somebody explain me why the CI is red please? 
I don't understand :thinking:

**Comment by mcornella** [comment]
It's an issue on GitHub's end, you can ignore it.

File: plugins/git/README.md [hunk 1]
```diff
@@ -131,7 +131,9 @@ plugins=(... git)
 | grmc                 | git rm --cached                                                                                                               |
 | grmv                 | git remote rename                                                                                                             |
 | grrm                 | git remote remove                                                                                                             |
+| grs                  | git restore                                                                                                                   |
 | grset                | git remote set-url                                                                                                            |
+| grs                  | git restore --source                                                                                                          |
```
Comment (mcornella) [inline plugins/git/README.md]: Should read `grss`

**Comment by mcornella** [comment]
I'm hesitant to allow these yet since [they're marked as experimental](https://github.blog/2019-08-16-highlights-from-git-2-23/#experimental-alternatives-for-git-checkout).

----
Nevermind, it looks like the names are here to stay, only the parameters *could* change: https://github.com/git/git/commit/4e43b7ff1ea4b6f16b93a432b6718e9ab38749bd.

File: plugins/git/README.md (hunk 1 above)
Comment (jjanvier) [reply plugins/git/README.md]: fixed!

**Comment by jjanvier** [comment]
@mcornella we can wait a little bit if you prefer. As you wish, no problem :)

The 2.23.0 has been officially released a few days ago https://github.com/git/git/releases.
So I guess, it _shouldn't_ change.

**Comment by jjanvier** [comment]
Thanks @mcornella :)

**Pull request merged**